packages/sources/kaiko/README.md

```diff
@@ -1,124 +1,52 @@
-        "ethx:spot:eth-usd",
-        "exxa:spot:eth-usd",
-        "gmni:spot:eth-usd",
-        "gacn:spot:eth-usd",
-        "itbi:spot:eth-usd",
-        "kcon:spot:eth-usd",
-        "krkn:spot:eth-usd",
-        "okcn:spot:eth-usd",
-        "lmax:spot:eth-usd",
-        "yobt:spot:eth-usd",
-        "quon:spot:eth-usd",
-        "stmp:spot:eth-usd",
-        "tbit:spot:eth-usd",
-        "wexn:spot:eth-usd"
-      ],
-      "start_timestamp": 1587140953277
-    },
-    "time": "2020-04-17T16:45:53.336Z",
-    "timestamp": 1587141953336,
-    "data": [
-      {
-        "timestamp": 1587141900000,
-        "price": "170.5073255331914"
-      },
-      {
-        "timestamp": 1587141600000,
-        "price": "170.64267792770045"
-      },
-      {
-        "timestamp": 1587141300000,
-        "price": "170.42193163824737"
-      },
-      {
-        "timestamp": 1587141000000,
-        "price": "170.55189478953133"
-      }
-    ],
-    "result": "170.5073255331914",
-    "access": {
-      "access_range": {
-        "start_timestamp": null,
-        "end_timestamp": 1606495255000
-      },
-      "data_range": {
-        "start_timestamp": 1572912000000,
-        "end_timestamp": null
-      }
-    }
-  },
-  "result": "170.5073255331914",
-  "statusCode": 200
-}
-```
+| Required? |      Name       |    Aliases     |                                                      Description                                                      |  Type  | Options |  Default   | Depends On | Not Valid With |
+| :-------: | :-------------: | :------------: | :-------------------------------------------------------------------------------------------------------------------: | :----: | :-----: | :--------: | :--------: | :------------: |
+|    ✅     |      base       | `coin`, `from` |                                    The symbol of symbols of the currency to query                                     | string |         |            |            |                |
+|    ✅     |      quote      | `market`, `to` |                                       The symbol of the currency to convert to                                        | string |         |            |            |                |
+|           |    interval     |                | The time interval to use in the query. NOTE: Changing this will likely require changing `millisecondsAgo` accordingly | string |         |    `2m`    |            |                |
+|           | millisecondsAgo |                |            Number of milliseconds from the current time that will determine start_time to use in the query            | string |         | `86400000` |            |                |
+|           |      sort       |                |                                   Which way to sort the data returned in the query                                    | string |         |   `desc`   |            |                |
+
+### Example
+
+There are no examples for this endpoint.
```

Comment:
Could we manually add an example, from the previous sample? I believe it will then be included by the generation script.